app/models/comment.rb

```diff
@@ -283,10 +283,19 @@ def current_upvoted?
 
   def delete_for_user(user, reason = nil)
     Comment.record_timestamps = false
```

Comment:
```suggestion
```

These `record_timestamps` lines were outdated. You couldn't know it, but Comment used to use `updated_at` to record the last time a comment was edited. We shifted to use a new `last_edited_at` column because Rails uses `updated_at` for clearing the cache anytime a record was changed for any reason... which we certainly want to do when a comment is deleted.